src_c/music.c

```diff
@@ -323,7 +390,10 @@ music_load(PyObject *self, PyObject *args)
 #if IS_SDLv1
             new_music = Mix_LoadMUS_RW(rw);
 #else  /* IS_SDLv2 */
-            new_music = Mix_LoadMUS_RW(rw, SDL_TRUE);
+            if (namehint)
+                new_music = Mix_LoadMUSType_RW(rw, _get_type_from_hint(namehint), SDL_TRUE);
```

Comment:
if _get_type_from_hint returns MUS_NONE, should we still call Mix_LoadMUSType_RW?